pkgs/development/python-modules/pip-chill/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, fetchFromGitHub, buildPythonPackage, bandit, coverage, flake8, nose
+, pytest, sphinx,
+#ssort,
+tox, twine, watchdog, python3, bump2version, pip, setuptools, pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pip-chill";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rbanffy";
+    repo = "pip-chill";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-oWq3UWBL5nsCBUkaElashZKvm7pN3StJNubgU++8YFs=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pip
+    bandit
+    coverage
+    flake8
+    nose
+    pytest
+    sphinx
+    #ssort
+    tox
+    twine
+    watchdog
+  ] ++ [ bump2version ];
```

Many of the packages you set as dependencies are tools dev-dependencies, and are not related to the package itself. `pytest` is also provided in check phase by the `pytestCheckHook`, so it shouldnt be listed too.

```suggestion
  dependencies = [ pip ];
```